Runtime/Model/Attributes/RuntimeAttributeProvider.cs

```diff
@@ -22,6 +22,7 @@ public void GetAttributes(IDictionary<string, string> attributes)
             attributes["application.company.name"] = Application.companyName;
             attributes["application.data_path"] = Application.dataPath;
             attributes["application.id"] = Application.identifier;
+            attributes["application.package"] = Application.identifier;

```

Comment:
is this supposed to have the same value as `"application.id"`?

Reply:
yeah